Marlin/Configuration.h

```diff
@@ -1460,4 +1460,26 @@
   //#define FILAMENT_LCD_DISPLAY
 #endif
 
+/** 
+  //=========================================================================
+  //================ DHT11, DHT21 or DHT22 Sensor Readings ==================
+  //=========================================================================
+ * 
+ * This feature is for reading the ambient temperature and humidity near or
+ * around the printer.
+ *
+ *         *******************************************************
+ *         =========   This will need to be installed!  ==========
+ *         *******************************************************
+ *         DHT Temperature & Humidity Sensor library for Arduino.
+ *         http://www.github.com/markruys/arduino-DHT
+ *         *******************************************************
+ * */
+#define DHT_ENABLE
+#if ENABLED(DHT_ENABLE)
+    #define DHT_TYPE 22     //DHT type 11, 21, or 22
+    #define DHT_PIN A3      //Set pin for the DHT sensor. Default = A3 (AUX1 connector pin3 (outside row))
+
+#endif //DHT_ENABLE
+
```

Comment:
There are much more Configuration.h-files in the `example_configurations` directory. All of them have to be modified.

Maybe `Configuration_adv.h` would be the better place for this feature.